### PR TITLE
[DEVHUB-1428]: Add Sort By To Top Level Pages

### PR DESCRIPTION
**Pull request by justinmckenzie**
## Jira Ticket:

Link to jira ticket [[DEVHUB-1428](https://jira.mongodb.org/browse/DEVHUB-1428)]

## Description:

Describe your changes in detail so for example expected behavior, actual behavior, steps tp reproduce the behavior

## Related Issue(s) or Dependencies (if applicable):

Please link to the issue(s) or dependencies here

## Checklist: (Check off where applicable)?

-   [ ] I have performed a self-review(QA) of my code and reviewed with Design or Product (If Applicable)?
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added tests that prove my fix is effective or that my feature works

## Screenshots (if appropriate):

[Recordit](https://recordit.co/) app or [Quicktime](https://apple.co/2J1EWUD) screen recorder

### CODE REVIEW

File: next.config.js
Comment (max-melo) [inline next.config.js:75]: Found an article which had images that weren't loading (https://www.mongodb.com/developer/products/mongodb/mongo-socket-chat-example/) while testing these changes, so I threw this in this PR.

File: src/components/search-filters/filter-group.tsx
Comment (max-melo) [inline src/components/search-filters/filter-group.tsx:198]: Adding a comment here for future reference:
Was seeing styling issues with new radio button component, figured out that it had to do with the lack of a global `box-sizing: border-box` declaration. @justinmckenzie and I discussed a solution that I'd hardcode the correct styles here and create a ticket to add this globally: https://jira.mongodb.org/browse/DEVHUB-1522.

This whole styling block should be removed once that ticket is completed.

File: package.json
Comment (justinmckenzie) [inline package.json:28]: @gfunk1230 are we ok with going to PROD with an RC version of Flora? They haven't technically approved/merged my [PR](https://github.com/10gen/flora/pull/104) yet, but gave us this so we can test out integration of the Radio component

**Review by ajhenri — APPROVED** [review]
LGTM, but +1 to @justinmckenzie's comment to check with Greg on the Flora version before merging.

File: package.json
Comment (gfunk1230) [reply package.json:28]: @justinmckenzie So let's not deploy a release candidate bc there may be things in there that they being flora, wants out in the wild yet. Like anything user-facing that affects the UI and UX. While I am sure we would test the app to make sure nothing breaks, what I am afraid will happen is that some team like marketing reached out to me and asked why our nav is inconsistent with what is on mongodb.com. Hope this all makes sense, so just to be safe than sorry, I rather hold off on deploying an RC version.

Comment (gfunk1230) [reply package.json:28]: So I asked Flora's team to make sure and they said they are cool with it. So if we ever get a ping from anyone saying there are any inconsistencies we can just mention we got the ok to release the RC version and that's why there are inconsistencies

**Comment by justinmckenzie** [comment]
> @justinmckenzie So let's not deploy a release candidate bc there may be things in there that they being flora, wants out in the wild yet. Like anything user-facing that affects the UI and UX. While I am sure we would test the app to make sure nothing breaks, what I am afraid will happen is that some team like marketing reached out to me and asked why our nav is inconsistent with what is on mongodb.com. Hope this all makes sense, so just to be safe than sorry, I rather hold off on deploying an RC version.

Marking this PR as `Do Not Merge` until we can safely release a stable Flora version.

**Pull request merged**
